repo/packages/M/mariadb/3/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion": "3.0",
+  "name": "mariadb",
+  "version": "10.1.21",
+  "scm": "https://github.com/MariaDB/server.git",
+  "maintainer": "kevin@fat.sh",
+  "website": "https://mariadb.org",
+  "description": "MariaDB is one of the most popular database servers in the world. It’s made by the original developers of MySQL and guaranteed to stay open source. Notable users include Wikipedia, Facebook and Google. MariaDB is developed as open source software and as a relational database it provides an SQL interface for accessing data. The latest versions of MariaDB also include GIS and JSON features.",
+  "tags": ["database", "mysql", "mariadb", "sql"],
+  "preInstallNotes": "In order for MariaDB service to start successfully it requires atleast 1 CPU and 1024MB of RAM including ports. WARNING: MariaDB on DCOS is currently in ALPHA. There may be bugs, incomplete\nfeatures, incorrect documentation, or other discrepancies.\n\nIf you didn't provide a value for `host_volume` in the CLI,\nYOUR DATA WILL NOT BE SAVED IN ANY WAY.\n",
```

Comment:
Remove `WARNING: MariaDB on DCOS is currently in ALPHA. There may be bugs, incomplete\nfeatures, incorrect documentation, or other discrepancies.` and this line at the beginning of `preInstallNotes`: `This DC/OS Service is currently in preview. There may be bugs, incomplete features, incorrect documentation, or other discrepancies. Preview packages should never be used in production!`